libs/stats/odc/stats/tasks.py

```diff
@@ -99,6 +99,66 @@ def __init__(
     def out_path(self, suffix: str) -> str:
         return out_path(suffix, self._output)
 
+    def get_dss(self,
```

Comment:
Is this meant to be internal function?
then start with `_`, `_get_dss`.

Some description of output is needed, particularly expected content of `cfg`

Reply:
Maybe it's time move that into dataclass of sorts.

Reply:
I actually had this, I just forgot to commit 🤦

Reply:
what do you mean by move into a data class?

Reply:
nah, let's leave it as is for now.

but what I mean is something like this thing:

https://github.com/opendatacube/odc-tools/blob/488d75f625226d6aefefbaa8f4ba45f64a7692da/libs/stats/odc/stats/model.py#L135-L145

basically just a "struct" that spells out what we expect from "cfg".